script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -1048,14 +1128,13 @@ She was just―――
 [sha:15dbc79a7fa91b738edf358b0494b388e738a716]{
 -- Page 112, Offset 29817.
 -- “―――だって、すごく楽しいんだもの。
-// line is from 10_00_ARC10_5
-// following two lines not voiced.
-"―――But, it was really fun.
+// This and subsequent offset from 10_00_ARC10_5, unvoiced
```

Comment:
I'd never take a script edit personally but comment edits? this is just crossing a line.

Reply:
Didn't notice that the old comment included following line 
![aichanteehee](https://user-images.githubusercontent.com/2476799/165002825-06b7d346-c20d-474c-9e7b-c5bb1aec96f3.png)